locations/spiders/cvs.py

```diff
@@ -0,0 +1,83 @@
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class CVSSpider(scrapy.Spider):
+
+    name = "cvs"
+    allowed_domains = ["www.cvs.com"]
+    start_urls = (
+        'https://www.cvs.com/store-locator/cvs-pharmacy-locations',
+    )
+
+    def parse_hours(self, response):
+        store_title = response.xpath('normalize-space(//h4[@class="nomargin store-photo-heading"]/text())').extract_first()
+        hours = {}
+        if store_title:
+            lis = response.xpath('//ul[@class="cleanList srHours srSection"]/li')
+            hours [store_title] = {}
+            for li in lis:
+                day = li.xpath('normalize-space(.//span[@class="day single-day"]/text() | .//span[@class="day"]/text())').extract_first()
+                times = li.xpath('.//span[@class="timings"]/text()').extract_first()
+                if day and times:
+                    hours [store_title].update({day: times})
+
+        phar_title = response.xpath('normalize-space(//h4[@class="printml"]/text())').extract_first()
```

Comment:
Please only include the store hours, not the hours for the pharmacy.

Reply:
👍